README.md

```diff
@@ -161,3 +161,21 @@ pre-commit install
 ```
 You're all set! Now when you commit code upstream the commit will trigger the different linters and flag the required changes
 Notes: you can quickly fix javascript eslint errors with `npm run lint:fix` 
+
+
+## Resetting the Database
+
+Every so often, it may be requested that the dev DB be "reset." Meaning, certain tables be truncated, and the data be reloaded. You can do that with the following set of commands. Note that these commands are for executing against a running docker container. If you are running the project in another way, adjust the incantations accordingly.
```

Comment:
It might be worth mentioning that this preserves user accounts, so any users who have signed up will still be able to login after the reset.

Reply:
Good idea. I've made more explicit.